classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,1215 @@
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method160(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method161(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method162(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method163(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method164(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method165(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method166(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method167(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method168(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method169(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method170(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method171(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method172(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method173(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method174(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method175(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method176(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method177(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method178(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method179(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method180(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method181(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method182(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method183(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method184(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method185(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method186(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method187(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method188(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method189(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method190(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method191(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method192(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method193(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method194(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method195(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method196(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method197(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method198(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method199(){
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.